tst/EnergyPlus/unit/ZoneTempPredictorCorrector.unit.cc

```diff
@@ -289,6 +296,7 @@ TEST_F(EnergyPlusFixture, ZoneTempPredictorCorrector_CorrectZoneHumRatTest)
     ZoneEquipConfig.deallocate();
     Node.deallocate();
     Zone.deallocate();
+    HybridModelZone.deallocate();
```

Comment:
Not necessary here. `HybridModel::clear_state()` will do this. In fact, most (all?) of these deallocates here aren't necessary.